### PR TITLE
Refactor css-modules to plain css

### LINKED ISSUE #195: Do not inject styles in page head!
Your library injects it's styles in page header with (probably) `addStylesToDom` function.

I use a separate css file with theme I need, so either *add option* to ignore style injecting, or provide a *separate css* file along with `dist/main.js`

## How it is

screenshot_2017-11-14_23-30-52

screenshot_2017-11-14_23-33-10

*(A lot of unnecessary css styles. `react-sortable-tree` is enabled)

## How it should be

screenshot_2017-11-14_23-38-13

*(No unnecessary css styles `react-sortable-tree` is disabled)

### PR DESCRIPTION
**Pull request by brendanmoore**
## Breaking Change
Users will now be required to manually import the style:
```javascript
import "react-sortable-tree/style.css";
```

### Notable Changes:
 - Styles object no-longer used.
 - Use `extract-text-webpack-plugin` for `dist` (examples/storybook remain unchanged).
 - Added `classnames.js` util to streamline `className` string concatenation.

Discussion ref #247 

Note: I am pretty confident the IE specific logic is sound but if someone can validate these changes work as expected on <IE10 that would be great.

Closes #195

### CODE REVIEW

File: src/node-renderer-default.js [hunk 1]
```diff
@@ -1,20 +1,23 @@
...
+const ROW_CLASS = oldIe ? 'rst__row rst__row_NoFlex' : 'rst__row';
+const ROW_CONTENTS_CLASS = oldIe
+  ? 'rst__rowContents rst__rowContents_NoFlex'
+  : 'rst__rowContents';
+const ROW_LABEL_CLASS = oldIe
+  ? 'rst__rowLabel rst__rowLabel_NoFlex'
+  : 'rst__rowLabel';
+const ROW_TOOLBAR_CLASS = oldIe
+  ? 'rst__rowToolbar rst__rowToolbar_NoFlex'
+  : 'rst__rowToolbar';
```
[the remark this answers is not in the document]
Comment (brendanmoore) [reply src/node-renderer-default.js]: Okay cool. I will remove.

**Comment by brendanmoore** [comment]
@fritz-c updates made

**Comment by brendanmoore** [comment]
I did update `README.md` with the extra line to import the style, I assume if pulled into master that will immediately reflect the "homepage" of this repo. Do you think it would be worth creating a new branch from master called "next"/"dev" or something?

**Comment by fritz-c** [comment]
Thanks for putting the time and effort into this! This is now available in v2.0.0

**Comment by brendanmoore** [comment]
No problem @fritz-c. You're welcome and great job getting out a v2.0.0!

**Pull request merged**